### DISCUSSION TITLE
Load data from API call from a specific column attribute of my livewire table

### DISCUSSION THREAD

**Discussion by lironsam** [livewire/livewire#2898] [1. Help]
Hi,

Thank you for this amazing package.

Using Livewire, I was able to build a datatable from my 'records' table.
Now, in this table, I have a column called "status" that is not included on my table 'records' of my Database.
This status is updated once in a while using an API call.

So how can I achieve this i.e for each row that is displayed call an API and update the cell ?

Thank you

**Comment by drbornot** [comment]
And why don't have a column in the db table for this??? how you think have persistent data if for example redirect to other route and go back again...make another API call??? you can get a job that update this status column periodically and always can get fresh data and persist between diff requests in the app. It's just my idea
